scripts/src/main/resources/scripts/command/ide

```diff
@@ -304,14 +304,14 @@ function doUninstall() {
 }
 
 function doUpdateProjects() {
-  for projectconfig in ${DEVON_IDE_HOME}/settings/projects/*.properties; do
+  for projectconfig in "${DEVON_IDE_HOME}"/settings/projects/*.properties; do
     echo "Importing from ${projectconfig}..."
-    local project_path=`cat $projectconfig | grep "^project.path" | cut -d'=' -f2`
-    local project_workingsets=`cat $projectconfig | grep "^project.workingsets" | cut -d'=' -f2`
-    local project_workspace=`cat $projectconfig | grep "^project.workspace" | cut -d'=' -f2`
-    local project_git_url=`cat $projectconfig | grep "^project.git.url" | cut -d'=' -f2`
-    local project_git_branch=`cat $projectconfig | grep "^project.git.branch" | cut -d'=' -f2`
-    local project_eclipse=`cat $projectconfig | grep "^project.eclipse" | cut -d'=' -f2`
+    project_path=$(grep "^project.path" "$projectconfig" | cut -d'=' -f2)
+    project_workingsets=$(grep "^project.workingsets" "$projectconfig" | cut -d'=' -f2)
+    project_workspace=$(grep "^project.workspace" "$projectconfig" | cut -d'=' -f2)
+    project_git_url=$(grep "^project.git.url" "$projectconfig" | cut -d'=' -f2)
+    project_git_branch=$(grep "^project.git.branch" "$projectconfig" | cut -d'=' -f2)
+    project_eclipse=$(grep "^project.eclipse" "$projectconfig" | cut -d'=' -f2)
```

Comment:
still declaring as local variables makes sense. I assume shellcheck was not complaining about the `local` keyword...

Reply:
Wasnt the `project.` prefix removed?

Reply:
Oh, OK, was done 2 commits later.

Reply:
Is `cut` command available on all platforms? I tend to use `cat`,`grep`, `sed`, and `awk` that are always present. Most probably `cut` is also fine, but we should verify that this also works in windows.

Reply:
Since I don't have Windows/Cygwin - I can't check this. Im pretty sure cut is standard like grep etc..

Reply:
Tested successfully in Windows VM:
```
 MINGW64 ~/cool-project/workspaces
cut --help
Usage: cut OPTION... [FILE]...
Print selected parts of lines from each FILE to standard output.
```